gen3/bin/kube-setup-pelicanjob.sh

```diff
@@ -9,26 +9,34 @@ gen3_load "gen3/gen3setup"
 
 hostname="$(g3kubectl get configmap global -o json | jq -r .data.hostname)"
 short_hostname=$(echo "$hostname" | cut -f1 -d".")
-bucketname="${short_hostname}-pfb-export"
-gen3 s3 create "$bucketname"
-awsuser="${short_hostname}-pelican"
-gen3 awsuser create "${short_hostname}-pelican"
-gen3 s3 attach-bucket-policy "$bucketname" --read-write --user-name "${short_hostname}-pelican"
+short_awsuser="${short_hostname}-pelican"
 
-mkdir -p $(gen3_secrets_folder)/g3auto/pelicanservice
-credsFile="$(gen3_secrets_folder)/g3auto/pelicanservice/config.json"
-if [[ (! -f "$credsFile") && -z "$JENKINS_HOME" ]]; then
-  gen3_log_info "initializing pelicanservice config.json"
-  user=$(gen3 secrets decode $awsuser-g3auto awsusercreds.json)
-  key_id=$(jq -r .id <<< $user)
-  access_key=$(jq -r .secret <<< $user)
-  cat - > "$credsFile" <<EOM
+if ! g3kubectl describe secret $short_awsuser-g3auto | grep awsusercreds.json > /dev/null 2>&1; then
```

Comment:
can we just use the same secret names in every commons, and different secret values?